tests/sentry/test_event_manager.py

```diff
@@ -587,7 +587,7 @@ def test_sdk(self):
     def test_no_message(self):
         # test that the message is handled gracefully
         manager = EventManager(self.make_event(**{
-            'message': '',
+            'message': None,
```

Comment:
This was a "fix" because the test for fcdd0cf42a751c758ef16007ea1d28cb404f6c8a was wrong.